code/modules/atmospherics/machinery/air_alarm/_air_alarm.dm

```diff
@@ -683,17 +692,25 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/airalarm, 27)
 ///Used for air alarm link helper, which connects air alarm to a sensor with corresponding chamber_id
 /obj/machinery/airalarm/proc/setup_chamber_link()
 	var/obj/machinery/air_sensor/sensor = GLOB.objects_by_id_tag[GLOB.map_loaded_sensors[air_sensor_chamber_id]]
-	if(isnull(sensor))
+	if(!sensor)
```

Comment:
Is there a reason in particular why we changed from `isnull` to `!`?

Reply:
I think it is a more common method in our codebase to check if an object exists or not.

The person who wrote this code was also using `!isnull()` in another place which is a double negative. Technically the `isnull` is a better boolean typecaste method, but I think it hurts readability.

Reply:
We have (largely) been moving to `isnull` for explicitness so moving back seems a little pointless

Reply:
Reverted via https://github.com/tgstation/tgstation/pull/86958/commits/213bb9e377b59773c789f59e55b1e6a9c31baf66

Last night I discovered people using `!!object` which has the same purpose as `!isnull(object)`. Nothing seems standardized so I guess it's more of a code smell.

Reply:
`!!object` is, specifically, casting to a boolean. Serves a different purpose than `!isnull(object)`.